maasglobal-json-schema-validator/README.md

```diff
@@ -8,10 +8,29 @@ import registry from 'maas-schemas/lib/ajv/registry';
 
 const validator = mjsv.validator([registry]);
 
+// eslint-disable-next-line deprecation/deprecation
+const validatorNonStrict = mjsv.validatorNonStrict([registry]);
+
 const phone = validator.validate(
   'https://schemas.maas.global/core/components/common.json#/definitions/phone',
   '+358401234567',
 );
+
+// Error thrown by validator
+const tuple = (): unknown =>
+  validator.validate(
+    'https://schemas.maas.global/core/components/non-strict-example.json',
+    ['one', 'two'],
+  );
+
+const nonStrictPhone = validatorNonStrict.validate(
```

Comment:
Feels a bit backwards to advertise a deprecated thing in the readme.

Reply:
Removed